veggieseasons/lib/widgets/search_bar.dart

```diff
@@ -17,40 +18,53 @@ class SearchBar extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return DecoratedBox(
-      decoration: BoxDecoration(
-        color: Styles.searchBackground,
-        borderRadius: BorderRadius.circular(10.0),
-      ),
-      child: Padding(
-        padding: const EdgeInsets.symmetric(
-          horizontal: 4.0,
-          vertical: 8.0,
-        ),
-        child: Row(
-          children: [
-            Icon(
-              CupertinoIcons.search,
-              color: Styles.searchIconColor,
-            ),
-            Expanded(
-              child: CupertinoTextField(
-                controller: controller,
-                focusNode: focusNode,
-                style: Styles.searchText,
-                cursorColor: Styles.searchCursorColor,
+    return BackdropFilter(
+      filter: ImageFilter.blur(sigmaX: 10, sigmaY: 10),
```

Comment:
There's a FrostedBox widget in `widgets/close_button.dart`. Could that be reused here?